SYCL/DeviceLib/built-ins/scalar_integer.cpp

```diff
@@ -363,26 +363,25 @@ int main() {
 
   // rotate (with large rotate size)
   {
-    s::cl_char r{0};
+    char r{0};
     {
-      s::buffer<s::cl_char, 1> BufR(&r, s::range<1>(1));
+      s::buffer<char, 1> BufR(&r, s::range<1>(1));
       s::queue myQueue;
       myQueue.submit([&](s::handler &cgh) {
         auto AccR = BufR.get_access<s::access::mode::write>(cgh);
         cgh.single_task<class rotateSI1SI2>([=]() {
-          AccR[0] = s::rotate(static_cast<s::cl_char>((unsigned char)0xe0),
-                              s::cl_char{50});
+          AccR[0] = s::rotate((unsigned char)0xe0, (unsigned char)50);
```

Comment:
not sure if it is important but arguments on the left is finally char, not unsigned ones

Reply:
Let's try to keep the test as close as possible to the original version, changed this line in 57be286a1fd89157c3f98a7054682b70d52b754f